browser-extension/mv3/src/service-worker/services/contextMenu.ts

```diff
@@ -37,14 +38,14 @@ export const initContextMenu = async () => {
   });
 
   chrome.contextMenus.onClicked.addListener(async (info) => {
+    const isExtensionStatusEnabled = await isExtensionEnabled();
     if (info.menuItemId === MenuItem.TOGGLE_ACTIVATION_STATUS) {
-      const isExtensionEnabled = await getVariable<boolean>(Variable.IS_EXTENSION_ENABLED, true);
-      setVariable<boolean>(Variable.IS_EXTENSION_ENABLED, !isExtensionEnabled);
+      setVariable<boolean>(Variable.IS_EXTENSION_ENABLED, !isExtensionStatusEnabled);
```

Comment:
@nafees87n Why are we setting the variables as opposite of the status here?

Reply:
It toggles the activation status of the extension

Reply:
Got it, BTW since we have exposed this functionality in the extension popup, we can safely remove it from contextMenu. Not a high priority just an FYI.